lib/warder/rails_advice_runner.rb

```diff
@@ -11,7 +11,22 @@ class RailsAdviceRunner < Runner
 
     def command_with_options
       path = @options.files.split(' ').first
-      "#{COMMAND_NAME} --silent --spec --test --features #{path}"
+      "#{COMMAND_NAME} --without-color --silent --spec --test --features #{path}"
+    end
+
+    def printable?(line)
+      super && !FAILURE_REGEXP.match(line)
+    end
+
+    def failed?(line)
+      match = FAILURE_REGEXP.match(line)
+      return false unless match
+      @stats_msg = line
+      !match[1].match('No ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.